google-cloud-graalvm-samples/graalvm-samples-client-library/datastore-sample/README.md

```diff
@@ -0,0 +1,53 @@
+# Datastore Sample Application with GraalVM
+
+This application uses the [Google Cloud Datastore client libraries](https://cloud.google.com/datastore/docs/reference/libraries) and is compatible with GraalVM compilation.
```

Comment:
"library"? We are only doing this for the Java client library.

Reply:
Done.